libs/cart/src/drivers/magento/transforms/outputs/cart-item.service.ts

```diff
@@ -29,9 +29,10 @@ export class DaffMagentoCartItemTransformer {
       product_id: String(cartItem.product.id),
 			image: {
 				id: cartItem.product.image.label,
-				url: cartItem.product.image.url, 
+				url: cartItem.product.image.url,
 				label: cartItem.product.image.label
-			},
+      },
+      discounted_row_total: cartItem.prices.row_total.value - cartItem.prices.total_item_discount.value,
```

Comment:
I'm pretty sure the total_item_discount is the total of discounts applied to an item without consideration of the quantity. The row_total does consider quantity.  We should probably just put a total_discounts on the DaffCartItem and return the discounted row total in a selector. That way, as we add more platforms, we don't need to keep doing this computation over and over (presuming other platforms also don't have a ready-made discounted_row_total field).

Reply:
Doing that computation in a selector seems better. Changed.